linux/keyman-config/keyman_config/__init__.py

```diff
@@ -68,6 +77,7 @@ def _(txt):
 
             SentryUrl = "https://1d0edbf2d0dc411b87119b6e92e2c357:e6d5a81ee6944fc79bd9f0cbb1f2c2a4@sentry.keyman.com/12"
             client = Client(SentryUrl, environment=__environment__, release=__version__)
+            client.user_context({'id': hash(getpass.getuser())})
```

Comment:
Is this still needed after l. 64?

Reply:
yes. L 64 gets only executed if we're using the new `sentry_sdk` module. But if that's not available (e.g. on Bionic) it throws an exception and we try the older `raven` module instead. That has a different way of setting the user.